oracle/src/test/scala/zio/sql/oracle/OracleModuleSpec.scala

```diff
@@ -0,0 +1,33 @@
+package zio.sql.oracle
+
+import zio.test.Assertion._
+import zio.test.TestAspect._
+import zio.test._
+
+import scala.language.postfixOps
+
+object OracleModuleSpec extends OracleRunnableSpec with ShopSchema {
```

Comment:
The tested module seems to be called `OracleSqlModule` so the test would be `OracleSqlModuleSpec`.

Reply:
It's merged so I will rename it within my change.

Reply:
:+1: